### PR TITLE
Add tags feed taxii

### PR DESCRIPTION
**Pull request by jochman**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/22895

## Minimum version of Demisto
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] No

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/FeedTAXII/Integrations/FeedTAXII/FeedTAXII.py [hunk 1]
```diff
@@ -1,9 +1,7 @@
 import tempfile
 from typing import Dict
 
-import demistomock as demisto
```
Comment (DeanArbel) [inline Packs/FeedTAXII/Integrations/FeedTAXII/FeedTAXII.py:4]: is this on purpose?

File: Packs/FeedTAXII/Integrations/FeedTAXII/FeedTAXII.py [hunk 2]
```diff
@@ -1,9 +1,7 @@
...
 from CommonServerPython import *
-from CommonServerUserPython import *
```
Comment (DeanArbel) [inline Packs/FeedTAXII/Integrations/FeedTAXII/FeedTAXII.py:6]: same here

File: Packs/FeedTAXII/Integrations/FeedTAXII/FeedTAXII.py [hunk 3]
```diff
@@ -939,13 +933,14 @@ def interval_in_sec(val):
...
     if isinstance(val, int):
         return val
-    range_split = val.split()
-    if len(range_split) != 2:
-        raise ValueError('Interval must be "number date_range_unit", examples: (2 hours, 4 minutes,6 months, 1 day.')
-    number = int(range_split[0])
-    range_unit = range_split[1].lower()
-    if range_unit not in ['minute', 'minutes', 'hour', 'hours', 'day', 'days']:
-        raise ValueError('The unit of Interval is invalid. Must be minutes, hours or days')
+    else:
```
Comment (DeanArbel) [inline Packs/FeedTAXII/Integrations/FeedTAXII/FeedTAXII.py:936]: This is feels more like a personal taste. Is this an enforced change?

File: Packs/FeedTAXII/Integrations/FeedTAXII/FeedTAXII.yml [hunk 4]
```diff
@@ -43,12 +43,13 @@ configuration:
   name: feedExpirationInterval
   required: false
   type: 1
-- defaultvalue: '240'
```
Comment (DeanArbel) [inline Packs/FeedTAXII/Integrations/FeedTAXII/FeedTAXII.yml:46]: was this intended?

File: Packs/FeedTAXII/Integrations/FeedTAXII/FeedTAXII.yml [hunk 5]
```diff
@@ -83,7 +84,6 @@ configuration:
   required: false
   type: 4
 - additionalinfo: Time (in seconds) before HTTP requests timeout.
-  defaultvalue: '20'
```
Comment (DeanArbel) [inline Packs/FeedTAXII/Integrations/FeedTAXII/FeedTAXII.yml:86]: please fix

File: Packs/FeedTAXII/Integrations/FeedTAXII/FeedTAXII.yml [hunk 6]
```diff
@@ -95,12 +95,16 @@ configuration:
   type: 0
 - additionalinfo: The time interval for the first fetch (retroactive). <number> <time
     unit> of type minute/hour/day. For example, 1 minute, 12 hours, 7 days.
-  defaultvalue: 1 day
```
Comment (DeanArbel) [inline Packs/FeedTAXII/Integrations/FeedTAXII/FeedTAXII.yml:98]: please fix

File: Packs/FeedTAXII/Integrations/FeedTAXII/FeedTAXII_test.py [hunk 7]
```diff
@@ -86,12 +88,21 @@ def test_interval_in_sec_6(self):
...
             raw_indicators = json.load(f)
             mocker.patch.object(client, 'build_iterator', return_value=raw_indicators)
             res = fetch_indicators_command(client)
-            with open('FeedTAXII_test/TestCommands/indicators_results.json', 'r') as exp_f:
+            with open('FeedTAXII_test/TestCommands/indicators_results.json') as exp_f:
                 expected = json.load(exp_f)
                 assert res == expected
+
+
+@pytest.mark.parametrize('tags', (['tags1, tags2'], []))
```
Comment (DeanArbel) [inline Packs/FeedTAXII/Integrations/FeedTAXII/FeedTAXII_test.py:101]: please add test docstrings in the new convention

File: Packs/FeedTAXII/Integrations/FeedTAXII/FeedTAXII.py (hunk 3 above)
Comment (jochman) [reply Packs/FeedTAXII/Integrations/FeedTAXII/FeedTAXII.py:938]: yeah, mypy or flake8 yelled on it.

**Pull request merged**